### PR TITLE
Bug 1843346: Removing CPUThrottlingHigh alert

### PR DESCRIPTION
**Pull request by paulfantom**
* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

/cc @lilic @simonpasquier

### CODE REVIEW

File: jsonnet/main.jsonnet [hunk 1]
```diff
@@ -41,6 +41,8 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
                        ruleGroup { rules: std.filter(function(rule) !('alert' in rule && rule.alert == 'etcdHighNumberOfFailedGRPCRequests'), ruleGroup.rules) }
                      else if ruleGroup.name == 'kubernetes-system' then
                        ruleGroup { rules: std.filter(function(rule) !('alert' in rule && rule.alert == 'KubeVersionMismatch'), ruleGroup.rules) }
+                     else if ruleGroup.name == 'kubernetes-resources' then
+                       ruleGroup { rules: std.filter(function(rule) !('alert' in rule && rule.alert == 'CPUThrottlingHigh'), ruleGroup.rules) }
```
Comment (simonpasquier) [inline jsonnet/main.jsonnet:45]: Maybe a comment with a link to the BZ. Also do we want to go one step further and remove all `info` alerts from upstream mixins by default?

Comment (brancz) [reply jsonnet/main.jsonnet:45]: That's not how I understood the conversation. I don't think we should be removing this or other info severity alerts.

Comment (lilic) [reply jsonnet/main.jsonnet:45]: No, this is only for this specific alert. I see no harm in info level alerts, what is your reasoning @simonpasquier behind it? (especially with docs explaining their purpose)

Comment (paulfantom) [reply jsonnet/main.jsonnet:45]: > Maybe a comment with a link to the BZ

It is linked in PR and we didn't do this for previous ones. Though I can add it if you want ;)

Comment (simonpasquier) [reply jsonnet/main.jsonnet:45]: > It is linked in PR and we didn't do this for previous ones. Though I can add it if you want ;)

I would argue that any dropped alert here should have a comment explaining why and with a link to the corresponding issue/bz. Using `git blame` to find out which PR introduced it and going back to the BZ isn't really friendly :)

Comment (paulfantom) [reply jsonnet/main.jsonnet:46]: Added comment ;)

**Pull request merged**